REPOSITORY_STANDARDS.md

```diff
@@ -0,0 +1,89 @@
+# Repository standards
+
+Here is a list of repository standards we have for our repositories. Much of the below is automated by our `dp-cli` tool for creating repositories but is documented here for completeness.
+
+## Contents
+
+- [naming conventions](#naming-conventions)
+- [visibility](#visibility)
+- [repository documentation](#repository-documentation)
+- [permissions](#permissions)
```

Comment:
Consider sentence case for the table of contents to be consistent with the headings.